package/yast2-network.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon May 18 12:59:27 CEST 2015 - locilka@suse.com
+
+- bsc#916376
+  - proposing iSCSI-related configuration in firewall proposal
+    in first stage if iSCSI is in use
+- 3.1.115
```

Comment:
Ah, true, that's because version have diverged and this is just a port from SLE 12. I forgot to raise the spec file version as well. Thanks!